include/wx/localedefs.h

```diff
@@ -108,6 +108,22 @@ struct WXDLLIMPEXP_BASE wxLanguageInfo
     const char* TrySetLocale() const;
 };
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent: contains platform dependent language name
+// ----------------------------------------------------------------------------
+
+class wxLocaleIdent
+{
+public:
+    wxLocaleIdent() = default;            // Leave name empty
+    wxLocaleIdent(const wxString& name);  // Constract name from wxString
```

Comment:
What is the format of `name`? I think it should be `language`, and there should be another constructor from `language` and `region`. And maybe also ctor from `wxLanguage`.

But if we have just a single ctor from a (full) name, there is not really much point in having this class at all...

Also, typo:
```suggestion
    wxLocaleIdent(const wxString& name);  // Construct name from wxString
```